### PR TITLE
Playground infraestructure (tests, etc) additions.

### PR DESCRIPTION
**Pull request by andrasio**
A few things to note:

* Nu can be started with a custom configuration file (`nu --config-file /path/to/sample_config.toml`). Useful for mocking the configuration on test runs.
* When given a custom configuration file Nu will save any changes to the file supplied appropiately.
* The `$nu.config-path` variable either shows the default configuration file (or the custom one, if given)
* We can now run end to end tests with finer grained control (currently, since this is baseline work, standard out) This will allow to check things like exit status, assert the contents with a format, etc)

### CODE REVIEW

File: crates/nu-command/src/commands/str_/to_integer.rs [hunk 1]
```diff
@@ -29,7 +28,7 @@ impl WholeStreamCommand for SubCommand {
         Signature::build("str to-int")
             .named("radix", SyntaxShape::Number, "radix of integer", Some('r'))
             .rest(
-                SyntaxShape::ColumnPath,
+                SyntaxShape::Any,
```
Comment (jonathandturner) [inline crates/nu-command/src/commands/str_/to_integer.rs:31]: Why switch to Any here?

Comment (andrasio) [reply crates/nu-command/src/commands/str_/to_integer.rs:31]: Oops, from another PR. Basically it allows to use `str to-int table_of_columns` so you can do either:

```
... | str to-int col1 col2 col3
... | str to-int $(= some_table)
```

Comment (andrasio) [reply crates/nu-command/src/commands/str_/to_integer.rs:31]: This is related to the issue/concern I brought up in the call the other day.

Comment (jonathandturner) [reply crates/nu-command/src/commands/str_/to_integer.rs:31]: Would you mind if we did that as a separate change?

Comment (andrasio) [reply crates/nu-command/src/commands/str_/to_integer.rs:31]: No problem.

**Pull request merged**